### PR TITLE
feat: add new authorizer cmds

### PR DESCRIPTION
**Pull request by iljaSL**

### CODE REVIEW

File: cmd/principals.go [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+}
+
+//
+//
+func principalSignCmd() *cobra.Command {
+	options := principalOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "sign",
+		Short: "Get a signature",
```
Comment (cuongssh) [inline cmd/principals.go:230]: Sign data using principal private key?

**Review by cuongssh — CHANGES_REQUESTED** [review]
The files are OK except webproxy lacks the configuration file donwload. 

It got a bit complicated as extender, web-proxy and carrier are trusted clients that also have data fetched through local-user-store micro-service. The List, Show sub-commands are actually list the CAs instead of showing  or listing the component's data.

I suggest, in addition to the missing conf download, the following:
- Save list and show sub-command for the calls to local-user-store to pull out the component data. Change the current list and show sub-command to list-ca and show-ca. 
- Similarly, revocation-list can be shortened to show-crl 
- webproxy consists of a web-proxy component and a carrier component. If the whole package is addressed by a single command "web-proxy"  then make sure it returns data from both components, or from a specific component if asked. If the whole package is addressed separately by "web-proxy" and "carrier" commands then add carrier.go

**Comment by iljaSL** [comment]
Restructured trusted client commands into one file -> extender.go
User can now choose with the `type` flag for a specific component.

File: cmd/extender.go [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+
+func init() {
+	rootCmd.AddCommand(extenderCmd())
+}
+
+//
+//
+func extenderCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "extender",
```
Comment (cuongssh) [inline cmd/extender.go]: Based on the usage it seems that the top-most command should be "clients" or "trusted-clients" as it covers extender, web-proxy and carrier now.

**Review by cuongssh — CHANGES_REQUESTED** [review]

**Comment by iljaSL** [comment]
Renamed the `extender` command to `trusted-clients`, as `clients` was already in use.
Renamed also the file, functions, variables etc. according to the command name change.

**Review by cuongssh — APPROVED** [review]

**Pull request merged**